src/config.ts

```diff
@@ -433,6 +433,11 @@ function migrateOldFormats(config: SBConfig) {
     if (config["previousVideoID"] !== undefined) {
         chrome.storage.sync.remove("previousVideoID");
     }
+
+    // populate invidiousInstances with new instances if 3p support is **DISABLED**
+    if (!config["supportInvidious"]) {
+        config["invidiousInstances"] = invidiousList
+    }
```

Comment:
```suggestion
    if (!config["supportInvidious"] && config["invidiousInstances"] !== invidiousList) {
        config["invidiousInstances"] = invidiousList;
    }
```

maybe this will save a call to chrome.storage.set each reload? Not sure if the equality will work though

Reply:
doesn't work, it modifies it every time

Reply:
maybe just a simple `.length` check?

Reply:
yea, probably good enough

Reply:
`.length` works like a charm